plasmapy/diagnostics/tests/test_thomson.py

```diff
@@ -510,8 +510,6 @@ def test_single_species_non_collective_spectrum(single_species_non_collective_sp
             ValueError,
             "All ions must be positively charged.",
         ),
-        # Value error when the ion list is empty
-        ({"ions": []}, ValueError, "At least one ion species needs to be defined."),
```

Comment:
A new pytest linter rule managed to find a bunch of duplicated tests when using `@pytest.mark.parametrize`.  Neat!